doc/classes/Callable.xml

```diff
@@ -187,7 +189,13 @@
 			<return type="Callable" />
 			<param index="0" name="argcount" type="int" />
 			<description>
-				Returns a copy of this [Callable] with the arguments unbound, as defined by [param argcount]. Calling the returned [Callable] will call the method without the extra arguments that are supplied in the [Callable] on which you are calling this method.
+				Returns a copy of this [Callable] with a number of arguments unbound. In other words, when the new callable is called the last few arguments supplied by the user are ignored, according to [param argcount]. The remaining arguments are passed to the callable. This allows to use the original callable in a context that attempts to pass more arguments than this callable can handle, e.g. a signal with a fixed number of arguments. See also [method bind].
+				[b]Note:[/b] When this method is chained with other similar methods binds from later calls appear before earlier calls.
```

Comment:
I don't think it's clear what's later and what's earlier. Which is why I suggested to explicitly say "read from the right". And your following example demonstrates how to read it from the right exactly.

Reply:
True, using "right to left" to make it more clear what right means